lib/middleware/request-handler.ts

```diff
@@ -362,8 +353,8 @@ function onAuthorize(req: Interface.IOurRequest,
         return next(new restify.BadRequestError(errorString));
     }
 
-    req.blpSession.authorizeUser(req.body)
-        .then((identity: blpapi.Identity): void => {
+    req.blpSession.authorize(req.body).
```

Comment:
I think `token` should either be a property of `req.body` or as part of url paramaters.